### ISSUE TITLE
Get Guitar Pro Tab fails

### ISSUE THREAD

**Issue by barrychapman** [masterT/ultimate-guitar-scraper#21]
Is there something we have to do to get the gpro tab? We get a download link, but that fails when going to it.

I believe that you need to pass over the `session_id` also, but will check to make sure.

**Comment by barrychapman** [comment]
Nevermind, found the error:

at line 252 of utils.js, you call:

      tab.content.url = 'https://tabs.ultimate-guitar.com/tabs/download?id=' + $downloadId.val()

It should be:

      tab.content.url = 'https://tabs.ultimate-guitar.com/**tab**/download?id=' + $downloadId.val()

Hope this helps

**Comment by masterT** [comment]
Hi,

It doesn't work, thank you for the notice. 😄 

The URL should be 

```js
'https://tabs.ultimate-guitar.com/download/public/' + $downloadId.val()
```

I will fix it and add a test.

**Comment by barrychapman** [comment]
I didn't see /download/public. `/tab/download` works too

**Comment by masterT** [comment]
Would we give me an example?

For the TAB 

https://tabs.ultimate-guitar.com/n/nirvana/smells_like_teen_spirit_power_tab.htm

previously it returned:

https://tabs.ultimate-guitar.com/tabs/download?id=227063 ❌

with my fix it returns:

https://tabs.ultimate-guitar.com/download/public/227063 ✅

**Comment by masterT** [comment]
Fixed in `ultimate-guitar-scraper@0.4.1` 🎉
